web/profiles/joinup/config/install/core.entity_form_display.user.user.subscription_settings.yml

```diff
@@ -3,8 +3,20 @@ status: true
 dependencies:
   config:
     - core.entity_form_mode.user.subscription_settings
+    - field.field.user.user.field_last_password_reset
+    - field.field.user.user.field_password_expiration
+    - field.field.user.user.field_social_media
+    - field.field.user.user.field_user_business_title
+    - field.field.user.user.field_user_content
+    - field.field.user.user.field_user_family_name
+    - field.field.user.user.field_user_first_name
     - field.field.user.user.field_user_frequency
     - field.field.user.user.field_user_group_types
+    - field.field.user.user.field_user_nationality
+    - field.field.user.user.field_user_organisation
+    - field.field.user.user.field_user_photo
+    - field.field.user.user.field_user_professional_domain
+    - field.field.user.user.field_user_professional_profile
```

Comment:
This is a side-effect of re-exporting this data using Config Devel, not critical for this issue but I decided to keep it in for completeness. This has also moved to the profile for consistency with the other view modes.